detox/src/devices/AttachedAndroidDriver.js

```diff
@@ -0,0 +1,38 @@
+const _ = require('lodash');
+const AAPT = require('./android/AAPT');
+const Emulator = require('./android/Emulator');
+const AndroidDriver = require('./AndroidDriver');
+
+class AttachedAndroidDriver extends AndroidDriver {
+
+  constructor(client) {
+    super(client);
+
+    this.aapt = new AAPT();
```

Comment:
I just found out AAPT was in EmulatorDriver and not in AndroidDriver, pushed a change, please remove it as well.